CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 - `[jest-runtime]` Fix missing coverage when using negative glob pattern in `testMatch` ([#7170](https://github.com/facebook/jest/pull/7170))
 - `[*]` Ensure `maxWorkers` is at least 1 (was 0 in some cases where there was only 1 CPU) ([#7182](https://github.com/facebook/jest/pull/7182))
 - `[jest-runtime]` Fix transform cache invalidation when requiring a test file from multiple projects ([#7186](https://github.com/facebook/jest/pull/7186))
+- `[expect]` Improves the failing message for `toStrictEqual` matcher. ([#7224])
```

Comment:
please include the URL as well, so it's clickable outside of github 🙂

Reply:
Oops. Small hickup. Will do!